env.js

```diff
@@ -1,11 +1,11 @@
 var dev =  {
   host: 'localhost',    // The database app server host
   port: 8000,           // By default port 8000 is enabled
-  //database : 'Documents',
-  user: 'admin',       // A user with at least the rest-writer role
-  password: 'admin',   // Probably not your password
-  authType: 'DIGEST'    // The default auth
-};
+  //database : "Documents",
```

Comment:
Don't commit your local password. You can modify this one locally, just don't commit it. (We probably need a better way to handle this.) Please modify back to its original state.